.github/workflows/common.yaml

```diff
@@ -33,6 +33,9 @@ jobs:
           go mod download
           make vendor pre-build build
         working-directory: ${{env.GOPATH}}/src/github.com/insolar/assured-ledger/ledger-core/v2
+      - name: Check if working directory is clean after build
+        run: scripts/gitstatus.sh
+        working-directory: ${{env.GOPATH}}/src/github.com/insolar/assured-ledger/ledger-core/v2
```

Comment:
I wonder  could we set 
```if: always()```
for this and lint rules, so we run tests even if linting/generate fails. To be able to faster see all the problems

Reply:
Let's wait for the first situation where we really need it